README.md

```diff
@@ -4,6 +4,8 @@ This repository contains UI related bits and pieces for Lightbend Cinnamon.
 
 ## Grafana dashboards
 
+** NOTE: As of version 2.10+ of Cinnamon dashboards are no longer published to this repository. Instead, you can [download the dashboards here](https://developer.lightbend.com/docs/cinnamon/current//visualizations/grafana.html#download-dashboards). **
```

Comment:
`current//visualizations`?

Reply:
I wonder how that got there?!? I'll update.